autoload/slime/targets/neovim.vim

```diff
@@ -105,12 +229,147 @@ function! s:translate_pid_to_id(pid)
   return -1
 endfunction
 
-function! s:translate_id_to_pid(id)
+function! s:translate_id_to_pid(id) abort
   let pid_out = -1
   try
     let pid_out = jobpid(a:id)
-  catch /E900: Invalid channel id/
-    let pid_out = -1
+  catch
   endtry
   return pid_out
 endfunction
+
+" Transforms a channel dictionary with job id and pid into an newline separated string  of job IDs.
+" for the purposes of input completion
+function! Last_channel_to_jobid(ArgLead, CmdLine, CursorPos) abort
+  let jobids = map(copy(g:slime_last_channel), {_, val -> val["jobid"]})
+  call map(jobids, {_, val -> string(val)})
+  return reverse(jobids) " making correct order in menu
+endfunction
+
+" Transforms a channel dictionary with job ida and pid into an newline separated string  of job PIDs.
```

Comment:
```suggestion
" Transforms a channel dictionary with job id and pid into a newline separated string of job PIDs.
```

elsewhere you use `jobid` (consistence)

Reply:
made `job ID` the standard throughout my contributions